pkg/rpcclient/wsclient_test.go

```diff
@@ -180,7 +182,7 @@ func TestWSClientEvents(t *testing.T) {
 	wsc.cache.initDone = true // Our server mock is restricted, so perform initialisation manually.
 	wsc.cache.network = netmode.UnitTestNet
 	wsc.cacheLock.Unlock()
-
+	close(startSending)
```

Comment:
Not here, it should be moved down below after subscriptions are being performed.